src/index.ts

```diff
@@ -74,6 +89,21 @@ export class LocalExec extends Resource {
           }),
         },
       },
-    ]);
+    ];
+
+    if (typeof this.onDestroy === "string") {
+      overrides.push({
+        "local-exec": {
+          when: "destroy",
+          working_dir: workingDir,
+          command: Lazy.stringValue({
+            // TODO: wrap command to capture stdout and stderr
```

Comment:
This TODO is left :)

Reply:
Yes, I copied it from the existing code.. IMHO it should be a separate PR to fix that issue: https://github.com/cdktf/cdktf-local-exec/pull/228/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R87

Reply:
Oh yeah, I remember wanting to spend some time on this topic 👍 Alright, let me merge this then 👍